front/app/modules/commercial/analytics/admin/messages.ts

```diff
@@ -0,0 +1,36 @@
+import { defineMessages } from 'react-intl';
+
```

Comment:
Created a new messages file in the root of the module, because these are used across components and hooks